scripts/twister

```diff
@@ -1164,8 +1170,16 @@ def main():
             start = ((subset - num_extra_sets - 1) * per_set) + base
             end = start + per_set
 
-        sliced_instances = islice(suite.instances.items(), start, end)
-        suite.instances = OrderedDict(sliced_instances)
+        sliced_instances = islice(to_run.items(), start, end)
+        if subset == 1:
```

Comment:
does it make more sense to add this to the last subset instead (since we already have the first subset in CI normally having the additional bsim work).

Reply:
this does not add any overhead, it just gets added for accounting and reporting. Either will work though.
Now that you mention it, it is time to split bsim into its own action, similar to clang

Reply:
>  Now that you mention it, it is time to split bsim into its own action, similar to clang

Yes, was just think the exact same thing on bsim.  Now that we have docker container and template for such things with the clang workflow :).